numba/jitclass/boxing.py

```diff
@@ -102,9 +101,8 @@ def _specialize_box(typ):
             dct[field] = property(getter, setter)
     # Inject methods as class members
     for name, func in typ.methods.items():
-        if not name.startswith('_'):
-            getter = _generate_method(name, func)
-            dct[name] = getter
+        if not (name.startswith('__') and name.endswith('__')):
```

Comment:
Special methods are ignored in this PR.  In #1865, there will be https://github.com/sklam/numba/blob/8e57a380abb6aba4c3bb90d206e03b0074107bbd/numba/jitclass/boxing.py#L120